op-mode-definitions/show-log.xml.in

```diff
@@ -50,6 +50,39 @@
             </properties>
             <command>cat $(printf "%s\n" /var/log/messages* | sort -nr) | grep -e heartbeat -e cl_status -e mach_down -e ha_log</command>
           </leafNode>
+          <node name="conntrack">
+            <properties>
+              <help>Show log for Conntrack Events</help>
```

Comment:
```suggestion
              <help>Show log for conntrack events</help>
```

There's no need to capitalize anything here: `conntrack` is consistently lowecase in netfilter manuals, and "Conntrack Events" is not a proper noun.